### PR TITLE
Workaround for issue #5738

### PR DESCRIPTION
**Pull request by abadams**
This disables the test on the specific target for which it is failing. The test is still run under opencl, but only on linux.

### CODE REVIEW

File: test/correctness/interpreter.cpp [hunk 1]
```diff
@@ -4,6 +4,14 @@ using namespace Halide;
 
 int main(int argc, char **argv) {
 
+    Target target = get_jit_target_from_environment();
```
Comment (steven-johnson) [inline test/correctness/interpreter.cpp:7]: Add `// TODO(#5738): remove after winbots are upgraded` or similar

**Review by steven-johnson — APPROVED** [review]

**Comment by abadams** [comment]
clean builds, woo. It's amazing what you can achieve by simply not running failing tests.

**Comment by steven-johnson** [comment]
for our next trick, we will increase the speed of apps/fft by 100x+ by simply filling in the results to zero and declaring it within acceptable error tolerance :-)

**Comment by abadams** [comment]
Hey, fast-math is on, so it's not a miscompilation

**Comment by steven-johnson** [comment]
more like `-fstupid-fast-math` than `-ffast-math` but sure

**Pull request merged**